### ISSUE TITLE
Errors in wptserve handlers aren't logged effectively

### ISSUE THREAD

**Issue by tommckee1** [web-platform-tests/wpt#30070]
tl;dr: wptserve should print stacktraces when request handlers fail

While working on crbug.com/1218565 I was hitting unexpected test failures that were ultimately caused by a Python error in one of our server-side request handlers (for the curious, https://github.com/web-platform-tests/wpt/blob/21f16b1e2f1b404e79d619c495585c922ee16e0e/resource-timing/resources/TAOResponse.py fails with an UnboundLocalError if the 'origin' request header is missing but the 'tao' query-parameter is set to 'match_origin').

When digging in, I found wptserve's stderr was empty but stdout contained something like
```
[<timestamp>,<pid> http on port 8001] ERROR - Exception loading http://www1.web-platform.test:8001/resource-timing/resources/TAOResponse.py?tao=match_origin&unique=8813844539736879: 500
[<timestamp>,<pid> http on port 8001] INFO - 500
```
I was expecting that, if a handler script threw an unhandled exception, there'd be some kind of stacktrace of what happened in the handler. This would give test authors a clear indication that they made a mistake in their request handler code.
